softdevice_controller/limitations.rst

```diff
@@ -6,8 +6,8 @@ Limitations
 When working with the SoftDevice Controller, you should be aware of the following limitations.
 In addition, see :ref:`nrf:known_issues` for temporary issues that will be fixed in future releases.
 
-YOPAN-87: The RSSI value reported by the SoftDevice Controller requires additional temperature compensation
-  The RSSI value reported by the SoftDevice Controller is the raw value from the radio peripheral.
+YOPAN-87: The RSSI value reported by the Softdevice Controller requires additional temperature compensation.
```

Comment:
why this changed?

Reply:
Ok, I see what happened. Fixes in documentation where introduced in this PR: https://github.com/nrfconnect/sdk-nrfxlib/pull/767 which was merged three days ago.

The doc was updated in sdk-nrfxlib, but there was no update in the original SDC repo, from which this PR is deployed. So until the doc is not updated in the original repo, the changes from the  https://github.com/nrfconnect/sdk-nrfxlib/pull/767 will always be overwritten.

Reply:
I will backport the changes to the original repo